Created Issue and PR Templates (#49)
Auto-DL/Generator: .github/PULL_REQUEST_TEMPLATE.md

> 
> ---
> name: PR template
> about: Submit a patch to fix issues and improve Auto-DL
> title: "[Fixes #] "
> labels: ''
> assignees: ''
> ---

Reviewer: ```suggestion
---
name: PR template
about: Submit a patch to fix issues and improve Auto-DL
title: "[Fixes #] "
labels: ''
assignees: ''

---
```

> ## Pull Request

Reviewer: ```suggestion

---
name: PR template
about: Submit a patch to fix issues and improve Auto-DL
title: "[Fixes #] "
labels: ''
assignees: ''
---

## Pull Request
```

> 

Reviewer: ```suggestion

fixes #  
<!-- Please add the issue number you're trying to fix with this PR, if none then please create an issue first -->
### Description
<!-- A line or two explaining what this patch does -->
```

> ### All Submissions:
> 
> - [ ] Have you followed the guidelines mentioned in our [Contributing document](/CONTRIBUTING.md)?
> - [ ] Have you ensured that there aren't other open [Pull Requests](../../../pulls) for the same update/change?
> - [ ] Is your [Pull Request](../../../pulls) linked to an [Issue](../../../issues)? If yes, please mention the Issue Number.

Reviewer: ```suggestion
### Checklist:
- [ ] Followed the guidelines mentioned in our [Contributing document](/CONTRIBUTING.md)
- [ ] Ensured that there aren't other open [Pull Requests](../../../pulls) for the same update/change
- [ ] This PR is linked to an [Issue](../../../issues) and the issue number has been mentioned in the title and body of the PR 
- [ ] The code is clean and docstrings have been added or updated as required
- [ ] Code is linted/formatted locally prior to submission (using `black`)
```

<!-- Please ensure you get maximum checks done before submitting a PR, for quick review and merge -->

> ### New Feature Submissions:
> 
> - [ ] Does your submission pass all `tests`?
> - [ ] Does your submission not lower the `coverage-rate`?
> - [ ] Have you run a linter through your code locally prior to submission, including `black` style?
> - [ ] Have you described your new feature in the `docs`?
> 
> ### Changes to Core Features:
> 
> - [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
> - [ ] Have you written new `tests` for your core changes, as applicable?
> - [ ] Have you successfully ran `tests` with your changes locally?
> - [ ] Have you kept the `coverage-rate` constant?
> - [ ] Have you managed to increase the `coverage-rate`?
> - [ ] Are the `docs` up to date?

Reviewer: ```suggestion
```
